.github/workflows/push-trigger.yml

```
@@ -94,4 +94,101 @@ jobs:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
+  build-apitest-keymanager-local:
+    needs: build-maven-apitest-keymanager
+    runs-on: ubuntu-latest
+    env:
+      NAMESPACE: ${{ secrets.dev_namespace_docker_hub }}
+      SERVICE_NAME: apitest-keymanager
+      SERVICE_LOCATION: api-test
+      BUILD_ARTIFACT: apitest-keymanager-local
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)